scripts/susan-wangari.js

```diff
@@ -5,4 +5,4 @@
     let emailAddress = "susanwangari810@gmail.com";
     let language = "JavaScript"
 
-console.log('Hello World, this is ' + firstName + " " + lastName + " " + 'with HNGi7 ID ' + id + " " + 'and email ' + emailAddress + " " + 'using' + " " + language + " " + 'for stage 2 task');
+console.log('Hello World, " " this is ' + firstName + " " + lastName + " " + 'with HNGi7 ID ' + id + " " + 'and email ' + emailAddress + " " + 'using' + " " + language + " " + 'for stage 2 task');
```

Comment:
this is the output it is displaying: `Hello World, " " this is Susan Wangari with HNGi7 ID HNG-02129 and email susanwangari810@gmail.com using JavaScript for stage 2 task`

run it on your browser console to check before you commit